### ISSUE TITLE
Difference between online and command line versions

### ISSUE THREAD

**Issue by nerd66** [hadolint/hadolint#1017]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

The `hadolint` command and the online version produce the same output for the same Dockerfile.

### Actual behavior

The provided Dockerfile produces no output with the `hadolint` command but the online version gives the following output for the second line: "Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>`" This shouldn't happen when using only a `requirements.txt` file.

### Steps to reproduce the behavior

Version: Haskell Dockerfile Linter 2.12.0
Save the below Dockerfile as `Dockerfile` and run `hadolint Dockerfile`. This produces no output.
Copy the Dockerfile content to the https://hadolint.github.io/hadolint/ and click on "Lint". This produces the above output about pinning pip package versions.

Minimal Dockerfile:

```Dockerfile
FROM python:3.10-slim
RUN pip install --no-cache-dir -Ur requirements.txt
```

**Comment by m-ildefons** [comment]
Online linter has been updated to latest release version (v2.14.0). Now it produces the same output as the command line version. Closing this issue.

<img width="2246" height="1527" alt="Image" src="https://github.com/user-attachments/assets/d2406491-2beb-4c4b-ab19-e2d3b01d2101" />
<img width="1786" height="207" alt="Image" src="https://github.com/user-attachments/assets/3da7ed4e-a163-4d68-aa30-ed252aec525a" />
